pkgs/by-name/bo/bounty/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  libgit2,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "bounty";
+  version = "0.1.0-unstable-25-01-29";
+
+  src = fetchFromGitHub {
+    owner = "ghbountybot";
+    repo = "cli";
+    rev = "a1c8e74b1373ef7e0cc0cef158eeec59d56623fc";
+    hash = "sha256-sLbEGLvupDWk5FBdu4UZ4ohqMUK/HKAHhpB3oJwRA0c=";
+  };
+
+  cargoHash = "sha256-q+lnEpTsNFuQYQlQGoxZ5vnFUTqGWRMqhK+20H9Ke0I=";
```

Comment:
Hash mismatch

Reply:
@RossComputerGuy isn't `cargoHash` for dependencies and `fetchFromGitHub` is for the source code? It seems to build for me.

Reply:
which command gave you a hash mismatch? Newto building from nixpkgs.

Reply:
`nix build`

Reply:
```
       > evaluation warning: rustPlatform.fetchCargoTarball is deprecated in favor of rustPlatform.fetchCargoVendor.
       >                     If you are using buildRustPackage, try setting useFetchCargoVendor = true and regenerating cargoHash.
       >                     See the 25.05 release notes for more information.
```
Eval fails still, please follow this.

Reply:
```
error: hash mismatch in fixed-output derivation '/nix/store/gik15b10vfmk12jz647hh4zrivd1hbig-bounty-0.1.0-unstable-25-01-29-vendor.tar.gz.drv':
         specified: sha256-q+lnEpTsNFuQYQlQGoxZ5vnFUTqGWRMqhK+20H9Ke0I=
            got:    sha256-9MrN4GLIhPAZBaDM/r8sXqlJQJmLwf1rdvnN8/aM1S4=
error: 1 dependencies of derivation '/nix/store/sdj3lz19jnd5vld4rk0npsliq6hniz4p-bounty-0.1.0-unstable-25-01-29.drv' failed to build
error: 1 dependencies of derivation '/nix/store/p1lsy2jsi389grgs54apliag3nxb49mi-review-shell.drv' failed to build
```

Reply:
fixed I think